pages/accounts/home.py

```diff
@@ -56,6 +69,44 @@ def login(self, user, password):
             self.find_element(*self._login_submit_button_locator).click()
             self.wait.until(lambda _: self.logged_in)
 
+        def facebook_login(self, user, facebook_user, password):
```

Comment:
Check through these sleeps...Would they make more sense to use a wait (`self.wait.until(...)`) before interacting with the elements?

Reply:
Updated!